tests/test_smoke.py

```diff
@@ -262,6 +262,24 @@ def test_image_id_dict_with_zone():
     run_one_test(test)
 
 
+def test_image_no_conda():
+    name = _get_cluster_name()
+    test = Test(
+        'image_no_conda',
+        [
+            # Use image id dict.
+            f'sky launch -y -c {name} examples/smoke_tests/no_conda_ami.yaml',
```

Comment:
How about placing the file in `tests/test_yamls/`?

Reply:
Yes good idea. I'm also thinking to move those test YAMLs to this directory as now our `/example` is pretty crowded. and those test YAMLs are usually just for testing purpose.